web_modules/layouts/Page/index.js

```diff
@@ -0,0 +1,70 @@
+import React, { PropTypes } from "react"
+import Helmet from "react-helmet"
+
+const Page = ({ head, body, __url }, context) => {
```

Comment:
no, look at the second parameter, i'm destructuring it in the function signature